README.md

```diff
@@ -1,2 +1,82 @@
 # queries
 Hosts reusable log queries which are built into a single queries.json file.
+
+
+# Query database structure
+
+```yaml
+queries:
+  query1:  # that is query id (sorted by id)
+    name: foo  # required
+    regex: bar  # optional
+    pattern: aaa # optional (either regex or pattern required)
+    url: ... # str or list[str], issue links to lp, bz,... (optional)
+    category: str  # required, one of ...
+    tags: []  # lists of tags (optional)
+    suppress-graph: false  # used for e-r (optional)
```

Comment:
this is completely not clear, "used for e-r"? it didn't help

Reply:
sure, will do.